modules/xmpp/ChatRoom.js

```diff
@@ -742,6 +769,19 @@ export default class ChatRoom extends Listenable {
             this.discoRoomInfo();
         }
 
+        // todo add a JSON layer such that not every json
+        // passed around in the chat is automatically assumed to be
+        // json used for messaging
+        const json = tryParseJSON(txt);
+
+        if (json) {
+            logger.log('chat json message', from, json);
```

Comment:
is it going to be many messages during regular session ? maybe debug level would be more appropriate here